java/src/main/java/api/example/java/Config.java

```diff
@@ -0,0 +1,89 @@
+package api.example.java;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.Base64Utils;
+import org.springframework.util.StringUtils;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@Configuration
+public class Config {
+
+	@Value("${climate.login.server}")
```

Comment:
For all java files: Suggest using 4 spaces instead of tab for indentation. Also, remove the extra blank lines.

Reply:
fixed